sdk/src/Services/DynamoDBv2/Custom/DataModel/AsyncSearch.cs

```diff
@@ -15,7 +15,6 @@
 
 using Amazon.DynamoDBv2.DocumentModel;
 using Amazon.Runtime.Telemetry.Tracing;
-using Amazon.Runtime.Telemetry.Tracing;
```

Comment:
For all DynamoDB changes here: Looks like some lines were added twice during the merge commit. I removed them and confirmed the contents of the `sdk/src/Services/DynamoDBv2/Custom` folder match what we had in `v4-development` before.